README.md

```diff
@@ -13,10 +13,14 @@ Made with ❤️ in Cluj-Napoca, Transylvania 🏰
 
 1. Clone the repo    
 1. Run `npm install`
-1. Serve it locally with `npm run serve`
-    * Running _production_ locally `npm run start-prod`
+1. Run
+    * `npm run copy-assets` 
+    * `npm run html-minify`
+    * `npm run bundle`
+    * `npm run build-sw`
```

Comment:
Instead of these lines, we could run the `npm run build-prod` and end up in the same place.